daras_ai_v2/base.py

```diff
@@ -2276,43 +2280,255 @@ def render_published_run_preview(self, published_run: PublishedRun):
         doc = published_run.saved_run.to_dict()
         self.render_example(doc)
 
-    def _render_example_preview(
-        self,
-        *,
-        published_run: PublishedRun,
-        allow_hide: bool,
-    ):
+    def render_published_run_full_width(self, published_run: PublishedRun, **kwargs):
+        hide_author_column = True if "hide_author_column" in kwargs else False
+        use_workspace_author = True if "use_workspace_author" in kwargs else False
+        max_desc_words = 230 if hide_author_column else 180
+        max_desc_words_mobile = 100
```

Comment:
Instead of this, you can use arguments with default values. That is more pythonic, clearer.

```python
def render_published_run_full_width(self, published_run: PublishedRun, show_author: bool = False, show_workspace: bool = False, show_workflow: bool = False, show_visibility: bool = False):
    max_description_words = 230 if hide_author_column else 100
    if show_author:
        ...
    if show_workflow:
        ...
    ....
```